### PR TITLE
Feature/#212

### PR DESCRIPTION
**Pull request by GalochkinKirov**
https://github.com/OAGr/foretold/issues/212

### CODE REVIEW

File: server/src/authorizers/agents.js [hunk 1]
```diff
@@ -16,12 +16,35 @@ async function currentAgentIsAuthenticatedRule(root, args, context, info) {
...
 }
 
+/**
+ * @param {*} root
+ * @param {object} args
+ * @param {Schema.Context} context
+ * @param {object} info
+ * @return {Promise<boolean>}
+ */
+async function currentAgentIsAdminRule(root, args, context, info) {
```
Comment (OAGr) [inline server/src/authorizers/agents.js]: can we call this ``currentAgentIsApplicationAdminRule`` or similar, and use similar terminology throughout this to distinguish ``applicationAdmin`` from ``channelAdmin``?

File: server/src/authorizers/permissions.js [hunk 2]
```diff
@@ -13,19 +14,24 @@ const { measurableIsOwnedByCurrentAgent } = require('./measurables');
...
+  currentAgentIsChannelAdmin,
+);
+
 const rulesChannel = {
   Query: {},
   Mutation: {
     channelUpdate: and(
       currentAgentIsAuthenticated,
-      currentAgentIsChannelAdmin,
+      currentAgentIsAdminOrChannelAdmin,
```
Comment (OAGr) [inline server/src/authorizers/permissions.js]: -> currentAgentIsApplicationAdminOrChannelAdmin

File: server/src/authorizers/agents.js (hunk 1 above)
Comment (GalochkinKirov) [reply server/src/authorizers/agents.js]: Updated

File: server/src/authorizers/permissions.js (hunk 2 above)
Comment (GalochkinKirov) [reply server/src/authorizers/permissions.js]: Thanks, updated too

**Comment by GalochkinKirov** [comment]
Thanks, updated

**Pull request merged**